packages/financial-templates-lib/test/clients/FinancialContractEventClient.js

```diff
@@ -63,6 +63,11 @@ const versionedIt = function (supportedVersions, shouldBeItOnly = false) {
   return runTestForVersion(supportedVersions, TESTED_CONTRACT_VERSIONS, iterationTestVersion) ? it : () => {};
 };
 
+const objectsInArrayInclude = (subset, superset) => {
```

Comment:
just want to make sure this is right. you have `subset, superset` here, but you always pass what looks like the superset first (array of poassible values), and the actual values second when calling this.

Reply:
It's actually the opposite I think. The values we explicitly write in the test may be a subset of the values produced by the implementation (aka the implementation may have more values than specified in the test and that's okay). Does that make sense? Or did I misunderstand your comment.

Reply:
yea that makes sense